fieldservice_fsm_person_salary_extended/views/fsm_order_salary_views.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_fsm_person_salary_extend_tree" model="ir.ui.view">
+        <field name="name">fsm.order.salary.tree (FSM Person Salary Extended)</field>
+        <field name="model">fsm.order.salary</field>
+        <field name="arch" type="xml">
+            <tree create="1" editable="bottom">
+                <field
+                    name="fsm_order_id"
+                    widget="many2onebutton"
+                    invisible="context.get('hide_fsm_order_id', True)"
+                />
+                <field name="sequence" widget="handle" />
+                <field name="fsm_person_id" />
+                <field
+                    name="location_id"
+                    widget="many2onebutton"
+                    invisible="context.get('hide_location_id', True)"
+                    optional="hide"
+                    />
+                <field
+                    name="sale_reference_salary"
+                    widget="many2onebutton"
+                    invisible="context.get('hide_sale_reference_salary', True)"
+                    optional="hide"
+                    />
+                <field name="vehicle" />
+                <field
+                    name="payments_salary" 
+                    groups="fieldservice.group_fsm_dispatcher"
+                    sum="Payments Salary" 
+                    />
+                <field
+                    name="payments_km"
+                    groups="fieldservice.group_fsm_dispatcher"
+                    sum="Payments Km" 
+                    />
+                <field 
+                    name="payments_diets"
+                    groups="fieldservice.group_fsm_dispatcher"
+                    sum="Payments Diets" 
+                    />
+
+                <field 
+                    name="payments_advance"
+                    groups="fieldservice.group_fsm_dispatcher"
+                    sum="Payments Advance" 
+                    />
+                <field 
+                    name="net_salary"
+                    groups="fieldservice.group_fsm_dispatcher"
+                    sum="Total" 
+                    />
+                <field name="currency_id" invisible= '1' />
```

Comment:
Please add `scheduled_date_start` header field to be shown in salary report view